tests/component-tests/textToSpeech/js/TextToSpeechTests.js

```diff
@@ -11,13 +11,40 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 */
 
 // Declare dependencies
-/* global fluid, jqUnit, speechSynthesis */
+/* global fluid, jqUnit, speechSynthesis, SpeechSynthesisUtterance */
 
 (function () {
     "use strict";
 
     fluid.registerNamespace("fluid.tests");
 
+    // Ensures that TTS is supported in the browser, including cases wehre the
+    // feature is detected, but where the underlying audio engine is missing.
+    // For example VM's on SauceLabs.
+    //
+    // returns a promise
```

Comment:
What does the returned promise represent? If this is intended to be a supported API, we should add a proper doc-style comment /** ... @param, @return etc. */ explaining the implementation and its strategy. What does the "delay" argument do?